docs/ref/templates/builtins.txt

```diff
@@ -842,13 +842,13 @@ output (as a string) inside a variable. This is useful if you want to use
 
 Regroups a list of alike objects by a common attribute.
 
-This complex tag is best illustrated by way of an example: say that "places" is
-a list of cities represented by dictionaries containing ``"name"``,
+This complex tag is best illustrated by way of an example: say that ``places``
+is a list of cities represented by dictionaries containing ``"name"``,
 ``"population"``, and ``"country"`` keys:
 
 .. code-block:: python
 
-    cities = [
+    places = [
```

Comment:
"places" was the name of the variable in the introduction paragraph (which was inconsistent with all the other examples).

Using "places" allows me to reuse the name "cities" in my `{% for country, cities in regrouped %}` examples (next commit) without it being too confusing.